config/hcp.go

```diff
@@ -154,11 +154,6 @@ func (c *hcpConfig) validate() error {
 		return fmt.Errorf("the SCADA address has to be non-empty")
 	}
 
-	// Ensure Profile is valid
-	if c.profile.GetOrganizationID() == "" || c.profile.GetProjectID() == "" {
-		return fmt.Errorf("organization ID and project ID must be non-empty")
-	}
-
```

Comment:
Let's keep these optional. But I would add some validation that ensures that if one of them has a value, the other one also has a value. Since they don't make sense without each other.